scripts/notes.ts

```diff
@@ -0,0 +1,239 @@
+import { execSync } from 'node:child_process';
+import chalk from 'chalk';
+import readline from 'readline';
+import themes from '../themes.json';
+import { writeFileSync } from 'node:fs';
+
+interface PRDetails {
+  author: string;
+  ghpr: number;
+  hash: string;
+  labels: string[] | null;
+  ticket: string | null;
+  title: string;
+}
+
+type Themes = typeof themes | any;
+const theme: Themes = themes;
+
+const c = chalk;
+
+const icons = {
+  breaking: '🚨',
+  added: '🚀',
+  changed: '💅',
+  fixed: '🛠',
+  internal: '📦',
+};
+
+type Icons = typeof icons | any;
+const icon: Icons = icons;
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+/**
+ * Current milestone reference
+ */
+const cm = JSON.parse(execSync('gh api repos/CMSgov/design-system/milestones').toString())[0];
```

Comment:
This would be a little clearer named something like `milestone`. You could still be pretty succinct accessing the data from it if you destructured it like

```suggestion
const milestone = JSON.parse(execSync('gh api repos/CMSgov/design-system/milestones').toString())[0];
const { title: milestoneTitle, open_issues, closed_issues } = milestone;
```